src/Traits/Categories/AppearanceHelpers.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+trait AppearanceHelpers
+{
+    /**
+     * Set the iOS badge increment count.
+     *
+     * @param int $count
+     *
+     * @return $this
+     */
+    public function incrementIosBadgeCount(int $count = 1)
+    {
+        return $this->setParameter('ios_badgeType', 'Increase')
+            ->setParameter('ios_badgeCount', $count);
+    }
+
+    /**
+     * Set the iOS badge decrement count.
+     *
+     * @param int $count
+     *
+     * @return $this
+     */
+    public function decrementIosBadgeCount(int $count = 1)
+    {
+        return $this->setParameter('ios_badgeType', 'Increase')
+            ->setParameter('ios_badgeCount', -1 * $count);
+    }
+
+    /**
+     * Set the iOS badge count.
+     *
+     * @param int $count
+     *
+     * @return $this
+     */
+    public function setIosBadgeCount(int $count)
+    {
+        return $this->setParameter('ios_badgeType', 'SetTo')
+            ->setParameter('ios_badgeCount', $count);
+    }
+
+    /**
+     * Set the iOS Sound.
+     *
+     * @param string $soundUrl
+     *
+     * @return $this
+     */
+    public function setIosSound(string $soundUrl)
```

Comment:
I kinda feel like "sound"  is not "appearance" ...maybe

Reply:
I've used the "categories" from the OneSignal Documentation for the "grouping" of the methods:
https://documentation.onesignal.com/reference
 And there are the sound Settings under "Appearance" :)

Reply:
oh - that makes way more sense haha - sorry should have checked that :) I like that 👍